dev/com.ibm.ws.messaging.common/resources/com/ibm/ws/sib/processor/CWSIPMessages.nlsprops

```diff
@@ -1069,9 +1069,9 @@ NO_NEIGHBOUR_REPLY_WARNING_CWSIP0381.useraction=Make sure that the remote messag
 
 # -------------------------------------------------------------------------------------------------
 # {0} The UUID of the Remote messaging engine
-NEIGHBOUR_REPLY_RECIEVED_INFO_CWSIP0382=CWSIP0382I: Messaging engine {0} responded to subscription request, Publish Subscribe topology now consistent.
-NEIGHBOUR_REPLY_RECIEVED_INFO_CWSIP0382.explanation=Successful communications have been made between messaging engines and the Publish subscribe Topology is consistent.
-NEIGHBOUR_REPLY_RECIEVED_INFO_CWSIP0382.useraction=none
+NEIGHBOUR_REPLY_RECEIVED_INFO_CWSIP0382=CWSIP0382I: Messaging engine {0} responded to subscription request, Publish Subscribe topology now consistent.
+NEIGHBOUR_REPLY_RECEIVED_INFO_CWSIP0382.explanation=Successful communications have been made between messaging engines and the Publish subscribe Topology is consistent.
```

Comment:
suggested edit:

Successful communications were made between messaging engines and the publish subscribe topology is consistent.

Reply:
I'm fine making this minor change, I'd prefer to keep the `publish subscribe` terminology as is. If you feel we should change this then I think an issue should be opened with the owning component to review their messages and ensure consistency. Really just trying to fix the blatant spelling mistakes here.